crates/trie/parallel/src/proof.rs

```diff
@@ -203,7 +203,11 @@ where
                     account.encode(&mut account_rlp as &mut dyn BufMut);
 
                     hash_builder.add_leaf(Nibbles::unpack(hashed_address), &account_rlp);
-                    storages.insert(hashed_address, storage_multiproof);
+
+                    // We might be adding leaves that are not necessarily our proof targets.
```

Comment:
why is it possible?

Reply:
because we don't store all of the branch nodes